{
    "type": "bundle",
    "id": "bundle--07353bc9-a2ec-4ae9-b25e-ea0cbce995dd",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--17133108-b7b7-4613-be6b-705dd7bdf296",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2022-09-17T00:58:51.214473Z",
            "modified": "2022-09-17T00:58:51.214473Z",
            "name": "CVE-2022-35982",
            "description": "TensorFlow is an open source platform for machine learning. If `SparseBincount` is given inputs for `indices`, `values`, and `dense_shape` that do not make a valid sparse tensor, it results in a segfault that can be used to trigger a denial of service attack. We have patched the issue in GitHub commit 40adbe4dd15b582b0210dfbf40c243a62f5119fa. The fix will be included in TensorFlow 2.10.0. We will also cherrypick this commit on TensorFlow 2.9.1, TensorFlow 2.8.1, and TensorFlow 2.7.2, as these are also affected and still in supported range. There are no known workarounds for this issue.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-35982"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}